### PR TITLE
feat: voiceConvertion

### PR DESCRIPTION
**Pull request by edouardnanodev**
Convert audio to the target voice

### CODE REVIEW

**Review by billNano — APPROVED** [review]
I want a demo

File: VoiceNanodev.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -420,21 +433,19 @@
...
 				CODE_SIGN_ENTITLEMENTS = VoiceNanodev/VoiceNanodev.entitlements;
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"VoiceNanodev/Preview Content\"";
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = VoiceNanodev/Info.plist;
-				INFOPLIST_KEY_CFBundleDisplayName = "Voice Nano";
```
Comment (royNanodev) [inline VoiceNanodev.xcodeproj/project.pbxproj:430]: Why is it removed?

**Review by royNanodev — CHANGES_REQUESTED** [review]

File: .DS_Store
Comment (royNanodev) [inline .DS_Store:1]: remove this file, and please add it to .gitIgnore

File: VoiceNanodev.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -391,21 +406,19 @@
...
```
Comment (royNanodev) [inline VoiceNanodev.xcodeproj/project.pbxproj:401]: samething here

File: VoiceNanodev.xcodeproj/project.xcworkspace/xcuserdata/edouard.xcuserdatad/UserInterfaceState.xcuserstate
Comment (royNanodev) [inline VoiceNanodev.xcodeproj/project.xcworkspace/xcuserdata/edouard.xcuserdatad/UserInterfaceState.xcuserstate:1]: what's the purpose of this file? needed?

**Review by royNanodev — CHANGES_REQUESTED** [review]
Looks like the convertion start only when we stop the recording.

**Review by royNanodev — APPROVED** [review]

**Pull request merged**